src/modules/Bootloader.rb

```diff
@@ -156,9 +156,20 @@ def Read
         ::Bootloader::BootloaderFactory.current.read
       rescue ::Bootloader::UnsupportedBootloader => e
         ret = Yast::Report.AnyQuestion(_("Unsupported Bootloader"),
-          _("Unsupported bootloader '%s' detected. Show proposal of supported configuration?") %
+          _("Unsupported bootloader '%s' detected. Use proposal of supported configuration instead?") %
             e.bootloader_name,
-          _("Show"),
+          _("Use"),
+          _("Quit"),
+          :yes) # focus proposing new one
+        return false unless ret
+
+        ::Bootloader::BootloaderFactory.current = ::Bootloader::BootloaderFactory.proposed
+        ::Bootloader::BootloaderFactory.current.propose
+      rescue ::Bootloader::BrokenConfiguration => e
```

Comment:
I don't have a full overview on how the pieces interact, but having this four lines here and also in bootloader/config_dialog.rb looks suspicious.